5] dice/main.py

```diff
@@ -0,0 +1,84 @@
+import random
+
+POSSIBLE_DICE = (
+
+    "D100",
+
+    "D20",
+
+    "D12",
+
+    "D10",
+
+    "D8",
+
+    "D6",
+
+    "D4",
+
+    "D3"
+
+)
+def roll_the_dice(dice_code):
+    """
+
+    Calculate dice roll from dice pattern.
+
+
+
+    :param str dice_code: dice pattern ex. `7D12-5`
+
+
+
+    :rtype: int, str
+
+    :return: dice roll value for proper dice pattern, `Wrong Input` text elsewhere
+
+    """
+
```

Comment:
tohle mas nejak divne naformatovane na to bacha. to skoro vypada jak kdyz si to odnekud zkopiroval a cely se ti to rozjelo. dost blbe se to mi to cte, tak to kdyztak pushni znovu s poradnym formtovanim